### DISCUSSION TITLE
js vs jsx?

### DISCUSSION THREAD

**Discussion by Nechir-89** [Tribalnetwork/Portfolio-Network#282] [Q&A]
Which one is better component extension to be used when developing a react web project?
- .js
- or jsx

@ZhongweiL 
@tmalik8 
@seansmith8912

**Comment by ZhongweiL** [comment]
I found this post talking about the same issue https://github.com/airbnb/javascript/pull/985, I think both sides are pretty convincing. Since there is no "correct" extension to use and the project has been using only .js extensions, I think we should keep it to avoid any potential issues.
